Swift/SwiftClosures.playground/Pages/Simple.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,23 @@
+var hiThere: () -> (String) = {
+    return "Hi there!"
+}
+hiThere()
+
+
+var square: (Int) -> (Int) = { x in
+    return x * x
+}
+
+/*: ###
+ Without explicitly declaring "x", you can use $0 as parameter:
+ */
+var squareAgain: (Int) -> (Int) = {
```

Comment:
@Inquisitor0 More imagination please. E. g. you may replace `squareAgain` with `multiply`.